pkgs/applications/audio/aacgain/default.nix

```diff
@@ -1,11 +1,12 @@
 { stdenv, fetchFromGitLab, fetchpatch }:
 
-stdenv.mkDerivation {
-  name = "aacgain-1.9.0";
+stdenv.mkDerivation rec {
```

Comment:
I'm a big :-1: on introducing more scopes (`rec`) just to avoid some tiny repetition.

Reply:
Without it, Nix just breaks. Why?